### PR TITLE
Settings  Screen

### PR DESCRIPTION
**Pull request by mateuszpmroz**

### CODE REVIEW

File: src/locale/en.ts [hunk 1]
```diff
@@ -56,4 +56,16 @@ export const en = {
     none: 'None',
     latest: 'Latest transaction',
   },
+  settings: {
```
Comment (bklukaczewski) [inline src/locale/en.ts:59]: locale in `src/locale/en` is essentially hardcoded - there's no mechanism to change it.
Use `i18n` and put your string values to `loc/en.js` :)

File: src/screens/Settings/LabeledSettingsRow.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  label: {
+    paddingBottom: 16,
+    ...typography.caption,
+    color: palette.textGrey,
+  },
+  greyLine: {
+    marginVertical: 24,
+    borderColor: palette.grey,
+    width: '120%',
+    borderWidth: 0.2,
```
Comment (bklukaczewski) [inline src/screens/Settings/LabeledSettingsRow.tsx:31]: Use `StyleSheet.hairlineWidth` as 0.2 points might not render at all for some devices

File: src/screens/Settings/LabeledSettingsRow.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+const styles = StyleSheet.create({
+  label: {
+    paddingBottom: 16,
+    ...typography.caption,
+    color: palette.textGrey,
+  },
+  greyLine: {
+    marginVertical: 24,
+    borderColor: palette.grey,
+    width: '120%',
```
Comment (bklukaczewski) [inline src/screens/Settings/LabeledSettingsRow.tsx:30]: This looks shady :) at least in the code. If it's looking good in the app then it's okay

File: src/screens/Settings/SettingsScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
+import AsyncStorage from '@react-native-community/async-storage';
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { NavigationScreenProps } from 'react-navigation';
+
+import { images, icons } from 'app/assets';
+import { Image, ScreenTemplate, Header, ListItem } from 'app/components';
+import { AppStorage, Biometric } from 'app/legacy';
+import { en } from 'app/locale';
```
Comment (bklukaczewski) [inline src/screens/Settings/SettingsScreen.tsx:9]:
```
import i18n from 'app/locale`

File: src/screens/Settings/SettingsScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+        onSwitchValueChange={onFingerprintLoginChange}
+        iconWidth={17}
+        iconHeight={19}
+      />
+    </>
+  );
+
+  const renderAboutSettings = () => <ListItem title={en.settings.aboutUs} source={icons.infoIcon} />;
+
+  return isLoading ? (
```
Comment (bklukaczewski) [inline src/screens/Settings/SettingsScreen.tsx:85]: you could just move 
```
if (isLoading) return null
```
to line 15

File: src/screens/Settings/LabeledSettingsRow.tsx (hunk 3 above)
Comment (mateuszpmroz) [reply src/screens/Settings/LabeledSettingsRow.tsx:30]: Yup, I've didn't found another way to set children element width higher than it's parent. This element exist in `ScreenTemplate` which has `padding: 20`. Another way could be passing styles by props to `ScreenTemplate` and change the padding to 0, then make some children padding 20 and some padding 0.

File: src/locale/en.ts (hunk 1 above)
Comment (mateuszpmroz) [reply src/locale/en.ts:59]: Yup, I know but I thought that we would implement it in the future :thinking: Okay so I'm gonna fix  :+1:

File: src/screens/Settings/SettingsScreen.tsx (hunk 5 above)
Comment (mateuszpmroz) [reply src/screens/Settings/SettingsScreen.tsx:85]: Wouldn't that return null always for that screen?

Comment (mateuszpmroz) [reply src/screens/Settings/SettingsScreen.tsx:85]: Wouldn't that return null always for that screen?

**Pull request merged**